test/util/src/main/java/org/apache/shardingsphere/test/loader/ExternalCaseSettings.java

```diff
@@ -15,18 +15,18 @@
  * limitations under the License.
  */
 
-package org.apache.shardingsphere.test.it.sql.parser.external;
+package org.apache.shardingsphere.test.loader;
 
 import java.lang.annotation.Inherited;
 import java.lang.annotation.Retention;
 import java.lang.annotation.RetentionPolicy;
 
 /**
- * External SQL parser integrate test settings.
+ * External SQL case test settings.
  */
 @Inherited
 @Retention(RetentionPolicy.RUNTIME)
-public @interface ExternalSQLParserITSettings {
+public @interface ExternalCaseSettings {
```

Comment:
How about create a new annotaion? It maybe not consistency with sql parser and IT